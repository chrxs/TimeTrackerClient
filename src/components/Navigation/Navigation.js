import React from 'react'
import PropTypes from 'prop-types'

import NavButton from 'components/NavButton'
import UserMenu from 'components/UserMenu'
import styles from './Navigation.scss'

const Navigation = ({
  currentUser,
  signOut
}) => {
  return (
    <nav className={styles.Navigation}>
      <ul>
        <li>
          <NavButton to='/' label='MY TIME' />
        </li>
        <li>
          <NavButton to='/people' label='PEOPLE' />
        </li>
        <li>
          <NavButton to='/projects' label='PROJECTS' />
        </li>
        <li>
          <div className={styles.NavButtons}>
            <UserMenu currentUser={currentUser} signOut={signOut} />
          </div>
        </li>
      </ul>
    </nav>
  )
}

Navigation.displayName = 'Navigation'

Navigation.propTypes = {
  currentUser: PropTypes.shape({
    name: PropTypes.string,
    image: PropTypes.string
  }).isRequired,
  signOut: PropTypes.func.isRequired
}

export default Navigation
