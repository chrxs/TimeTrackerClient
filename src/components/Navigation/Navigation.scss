@import '../../variables';

.Navigation {
  background-color: $brand-primary-color;
  margin: 0;
  padding: 15px 10px;
  color: #fff;

  ul {
    list-style-type: none;
    margin: 0;
    padding: 0;
    display: flex;
    flex-direction: row;
    justify-content: flex-start;
    align-items: flex-start;
  }

  li {
    margin: 0 14px;
    padding: 0;
  }

  li:last-child {
    margin-left: auto;
  }
}
