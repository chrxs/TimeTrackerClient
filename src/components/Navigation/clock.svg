<svg viewBox="0 0 62 63" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
  <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
    <path d="M36.0000429,17.1428096 L36.0000429,35.1428311 C36.0000429,35.8660462 35.4375422,36.4285469 34.7143271,36.4285469 L21.8571689,36.4285469 C21.1339538,36.4285469 20.5714531,35.8660462 20.5714531,35.1428311 L20.5714531,32.5713995 C20.5714531,31.8481843 21.1339538,31.2856836 21.8571689,31.2856836 L30.8571796,31.2856836 L30.8571796,17.1428096 C30.8571796,16.4195945 31.4196803,15.8570938 32.1428955,15.8570938 L34.7143271,15.8570938 C35.4375422,15.8570938 36.0000429,16.4195945 36.0000429,17.1428096 Z M52.7143486,31.2856836 C52.7143486,19.2320978 42.9107654,9.42851472 30.8571796,9.42851472 C18.8035938,9.42851472 9.00001073,19.2320978 9.00001073,31.2856836 C9.00001073,43.3392694 18.8035938,53.1428525 30.8571796,53.1428525 C42.9107654,53.1428525 52.7143486,43.3392694 52.7143486,31.2856836 Z M61.7143593,31.2856836 C61.7143593,48.3214182 47.8929142,62.1428633 30.8571796,62.1428633 C13.821445,62.1428633 0,48.3214182 0,31.2856836 C0,14.249949 13.821445,0.42850399 30.8571796,0.42850399 C47.8929142,0.42850399 61.7143593,14.249949 61.7143593,31.2856836 Z" fill="#FFFFFF"></path>
  </g>
</svg>
