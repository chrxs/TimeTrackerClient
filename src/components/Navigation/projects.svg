<svg viewBox="0 0 67 57" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
  <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
    <path d="M66.8572226,19.7142413 C66.8572226,14.7722711 62.799182,10.7142305 57.8572118,10.7142305 L30.8571796,10.7142305 L30.8571796,9.42851472 C30.8571796,4.48654454 26.7991391,0.42850399 21.8571689,0.42850399 L9.00001073,0.42850399 C4.05804055,0.42850399 0,4.48654454 0,9.42851472 L0,47.9999893 C0,52.9419594 4.05804055,57 9.00001073,57 L57.8572118,57 C62.799182,57 66.8572226,52.9419594 66.8572226,47.9999893 L66.8572226,19.7142413 Z" fill="#FFFFFF"></path>
  </g>
</svg>
