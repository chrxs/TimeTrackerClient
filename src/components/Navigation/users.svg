<svg viewBox="0 0 78 73" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
  <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
    <path d="M23.8259213,36.2856836 C19.6473449,36.4062195 15.8705546,38.2142574 13.1785871,41.4285469 L7.79465215,41.4285469 C3.77679022,41.4285469 3.941617e-16,39.4999732 3.941617e-16,35.0401464 C3.941617e-16,31.7856783 -0.120535858,20.8570938 4.9821488,20.8570938 C5.8258998,20.8570938 10.0044762,24.2722765 15.4285898,24.2722765 C17.2768063,24.2722765 19.0446656,23.9508475 20.7723462,23.3481682 C20.6518103,24.2320978 20.5714531,25.1160275 20.5714531,25.9999571 C20.5714531,29.6562114 21.7366331,33.2722872 23.8259213,36.2856836 Z M66.8572226,61.8794641 C66.8572226,68.3884005 62.5581103,72.2857265 56.1295312,72.2857265 L21.0134179,72.2857265 C14.5848388,72.2857265 10.2857265,68.3884005 10.2857265,61.8794641 C10.2857265,52.7990962 12.4151934,38.8571153 24.1875288,38.8571153 C25.5536019,38.8571153 30.5357507,44.4419434 38.5714746,44.4419434 C46.6071984,44.4419434 51.5893472,38.8571153 52.9554203,38.8571153 C64.7277557,38.8571153 66.8572226,52.7990962 66.8572226,61.8794641 Z M25.7143164,10.5713673 C25.7143164,16.2365526 21.0937751,20.8570938 15.4285898,20.8570938 C9.7634045,20.8570938 5.14286327,16.2365526 5.14286327,10.5713673 C5.14286327,4.90618194 9.7634045,0.285640717 15.4285898,0.285640717 C21.0937751,0.285640717 25.7143164,4.90618194 25.7143164,10.5713673 Z M54.0000644,25.9999571 C54.0000644,34.5178244 47.0893418,41.4285469 38.5714746,41.4285469 C30.0536073,41.4285469 23.1428847,34.5178244 23.1428847,25.9999571 C23.1428847,17.4820898 30.0536073,10.5713673 38.5714746,10.5713673 C47.0893418,10.5713673 54.0000644,17.4820898 54.0000644,25.9999571 Z M77.1429491,35.0401464 C77.1429491,39.4999732 73.3661589,41.4285469 69.348297,41.4285469 L63.964362,41.4285469 C61.2723945,38.2142574 57.4956043,36.4062195 53.3170278,36.2856836 C55.406316,33.2722872 56.571496,29.6562114 56.571496,25.9999571 C56.571496,25.1160275 56.4911388,24.2320978 56.3706029,23.3481682 C58.0982835,23.9508475 59.8661428,24.2722765 61.7143593,24.2722765 C67.1384729,24.2722765 71.3170493,20.8570938 72.1608003,20.8570938 C77.263485,20.8570938 77.1429491,31.7856783 77.1429491,35.0401464 Z M72.0000858,10.5713673 C72.0000858,16.2365526 67.3795446,20.8570938 61.7143593,20.8570938 C56.049174,20.8570938 51.4286327,16.2365526 51.4286327,10.5713673 C51.4286327,4.90618194 56.049174,0.285640717 61.7143593,0.285640717 C67.3795446,0.285640717 72.0000858,4.90618194 72.0000858,10.5713673 Z" fill="#FFFFFF"></path>
  </g>
</svg>
