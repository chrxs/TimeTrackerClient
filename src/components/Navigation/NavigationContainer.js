import { withRouter } from 'react-router-dom'
import { connect } from 'react-redux'

import { getCurrentUser } from 'state/currentUser/reducer'
import { signOut } from 'state/currentUser/actionCreators'

import Navigation from './Navigation'

const mapStateToProps = (state) => {
  return {
    currentUser: getCurrentUser(state)
  }
}

const mapDispatchToProps = (dispatch) => {
  return {
    signOut () {
      return dispatch(signOut())
    }
  }
}

export default withRouter(
  connect(
    mapStateToProps,
    mapDispatchToProps
  )(Navigation)
)
