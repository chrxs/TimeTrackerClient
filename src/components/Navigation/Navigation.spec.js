/* global describe, test, expect */
import React from 'react'
import { BrowserRouter } from 'react-router-dom'

import Navigation from './Navigation'
import renderer from 'react-test-renderer'

const currentUser = {
  image: 'http://www.image.jpg',
  name: 'Harry'
}
const history = { push: () => {} }
const signOut = () => new Promise().resolve()

describe('Navigation', () => {
  test('Navigation snapshot', () => {
    const component = renderer.create(
      <BrowserRouter>
        <Navigation currentUser={currentUser} history={history} signOut={signOut} />
      </BrowserRouter>
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
