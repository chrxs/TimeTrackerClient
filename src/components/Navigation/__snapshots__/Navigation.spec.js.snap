// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Navigation snapshot 1`] = `
<nav
  className="Navigation"
>
  <ul>
    <li>
      <a
        className="NavButton"
        href="/"
        onClick={[Function]}
        style={undefined}
      >
        <div
          className="icon"
          style={
            Object {
              "backgroundImage": "url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgNjIgNjMiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+ICA8ZyBzdHJva2U9Im5vbmUiIHN0cm9rZS13aWR0aD0iMSIgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj4gICAgPHBhdGggZD0iTTM2LjAwMDA0MjksMTcuMTQyODA5NiBMMzYuMDAwMDQyOSwzNS4xNDI4MzExIEMzNi4wMDAwNDI5LDM1Ljg2NjA0NjIgMzUuNDM3NTQyMiwzNi40Mjg1NDY5IDM0LjcxNDMyNzEsMzYuNDI4NTQ2OSBMMjEuODU3MTY4OSwzNi40Mjg1NDY5IEMyMS4xMzM5NTM4LDM2LjQyODU0NjkgMjAuNTcxNDUzMSwzNS44NjYwNDYyIDIwLjU3MTQ1MzEsMzUuMTQyODMxMSBMMjAuNTcxNDUzMSwzMi41NzEzOTk1IEMyMC41NzE0NTMxLDMxLjg0ODE4NDMgMjEuMTMzOTUzOCwzMS4yODU2ODM2IDIxLjg1NzE2ODksMzEuMjg1NjgzNiBMMzAuODU3MTc5NiwzMS4yODU2ODM2IEwzMC44NTcxNzk2LDE3LjE0MjgwOTYgQzMwLjg1NzE3OTYsMTYuNDE5NTk0NSAzMS40MTk2ODAzLDE1Ljg1NzA5MzggMzIuMTQyODk1NSwxNS44NTcwOTM4IEwzNC43MTQzMjcxLDE1Ljg1NzA5MzggQzM1LjQzNzU0MjIsMTUuODU3MDkzOCAzNi4wMDAwNDI5LDE2LjQxOTU5NDUgMzYuMDAwMDQyOSwxNy4xNDI4MDk2IFogTTUyLjcxNDM0ODYsMzEuMjg1NjgzNiBDNTIuNzE0MzQ4NiwxOS4yMzIwOTc4IDQyLjkxMDc2NTQsOS40Mjg1MTQ3MiAzMC44NTcxNzk2LDkuNDI4NTE0NzIgQzE4LjgwMzU5MzgsOS40Mjg1MTQ3MiA5LjAwMDAxMDczLDE5LjIzMjA5NzggOS4wMDAwMTA3MywzMS4yODU2ODM2IEM5LjAwMDAxMDczLDQzLjMzOTI2OTQgMTguODAzNTkzOCw1My4xNDI4NTI1IDMwLjg1NzE3OTYsNTMuMTQyODUyNSBDNDIuOTEwNzY1NCw1My4xNDI4NTI1IDUyLjcxNDM0ODYsNDMuMzM5MjY5NCA1Mi43MTQzNDg2LDMxLjI4NTY4MzYgWiBNNjEuNzE0MzU5MywzMS4yODU2ODM2IEM2MS43MTQzNTkzLDQ4LjMyMTQxODIgNDcuODkyOTE0Miw2Mi4xNDI4NjMzIDMwLjg1NzE3OTYsNjIuMTQyODYzMyBDMTMuODIxNDQ1LDYyLjE0Mjg2MzMgMCw0OC4zMjE0MTgyIDAsMzEuMjg1NjgzNiBDMCwxNC4yNDk5NDkgMTMuODIxNDQ1LDAuNDI4NTAzOTkgMzAuODU3MTc5NiwwLjQyODUwMzk5IEM0Ny44OTI5MTQyLDAuNDI4NTAzOTkgNjEuNzE0MzU5MywxNC4yNDk5NDkgNjEuNzE0MzU5MywzMS4yODU2ODM2IFoiIGZpbGw9IiNGRkZGRkYiPjwvcGF0aD4gIDwvZz48L3N2Zz4=)",
            }
          }
        />
        <div
          className="label"
        >
          MY TIME
        </div>
      </a>
    </li>
    <li>
      <a
        className="NavButton"
        href="/people"
        onClick={[Function]}
        style={undefined}
      >
        <div
          className="icon"
          style={
            Object {
              "backgroundImage": "url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgNzggNzMiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+ICA8ZyBzdHJva2U9Im5vbmUiIHN0cm9rZS13aWR0aD0iMSIgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj4gICAgPHBhdGggZD0iTTIzLjgyNTkyMTMsMzYuMjg1NjgzNiBDMTkuNjQ3MzQ0OSwzNi40MDYyMTk1IDE1Ljg3MDU1NDYsMzguMjE0MjU3NCAxMy4xNzg1ODcxLDQxLjQyODU0NjkgTDcuNzk0NjUyMTUsNDEuNDI4NTQ2OSBDMy43NzY3OTAyMiw0MS40Mjg1NDY5IDMuOTQxNjE3ZS0xNiwzOS40OTk5NzMyIDMuOTQxNjE3ZS0xNiwzNS4wNDAxNDY0IEMzLjk0MTYxN2UtMTYsMzEuNzg1Njc4MyAtMC4xMjA1MzU4NTgsMjAuODU3MDkzOCA0Ljk4MjE0ODgsMjAuODU3MDkzOCBDNS44MjU4OTk4LDIwLjg1NzA5MzggMTAuMDA0NDc2MiwyNC4yNzIyNzY1IDE1LjQyODU4OTgsMjQuMjcyMjc2NSBDMTcuMjc2ODA2MywyNC4yNzIyNzY1IDE5LjA0NDY2NTYsMjMuOTUwODQ3NSAyMC43NzIzNDYyLDIzLjM0ODE2ODIgQzIwLjY1MTgxMDMsMjQuMjMyMDk3OCAyMC41NzE0NTMxLDI1LjExNjAyNzUgMjAuNTcxNDUzMSwyNS45OTk5NTcxIEMyMC41NzE0NTMxLDI5LjY1NjIxMTQgMjEuNzM2NjMzMSwzMy4yNzIyODcyIDIzLjgyNTkyMTMsMzYuMjg1NjgzNiBaIE02Ni44NTcyMjI2LDYxLjg3OTQ2NDEgQzY2Ljg1NzIyMjYsNjguMzg4NDAwNSA2Mi41NTgxMTAzLDcyLjI4NTcyNjUgNTYuMTI5NTMxMiw3Mi4yODU3MjY1IEwyMS4wMTM0MTc5LDcyLjI4NTcyNjUgQzE0LjU4NDgzODgsNzIuMjg1NzI2NSAxMC4yODU3MjY1LDY4LjM4ODQwMDUgMTAuMjg1NzI2NSw2MS44Nzk0NjQxIEMxMC4yODU3MjY1LDUyLjc5OTA5NjIgMTIuNDE1MTkzNCwzOC44NTcxMTUzIDI0LjE4NzUyODgsMzguODU3MTE1MyBDMjUuNTUzNjAxOSwzOC44NTcxMTUzIDMwLjUzNTc1MDcsNDQuNDQxOTQzNCAzOC41NzE0NzQ2LDQ0LjQ0MTk0MzQgQzQ2LjYwNzE5ODQsNDQuNDQxOTQzNCA1MS41ODkzNDcyLDM4Ljg1NzExNTMgNTIuOTU1NDIwMywzOC44NTcxMTUzIEM2NC43Mjc3NTU3LDM4Ljg1NzExNTMgNjYuODU3MjIyNiw1Mi43OTkwOTYyIDY2Ljg1NzIyMjYsNjEuODc5NDY0MSBaIE0yNS43MTQzMTY0LDEwLjU3MTM2NzMgQzI1LjcxNDMxNjQsMTYuMjM2NTUyNiAyMS4wOTM3NzUxLDIwLjg1NzA5MzggMTUuNDI4NTg5OCwyMC44NTcwOTM4IEM5Ljc2MzQwNDUsMjAuODU3MDkzOCA1LjE0Mjg2MzI3LDE2LjIzNjU1MjYgNS4xNDI4NjMyNywxMC41NzEzNjczIEM1LjE0Mjg2MzI3LDQuOTA2MTgxOTQgOS43NjM0MDQ1LDAuMjg1NjQwNzE3IDE1LjQyODU4OTgsMC4yODU2NDA3MTcgQzIxLjA5Mzc3NTEsMC4yODU2NDA3MTcgMjUuNzE0MzE2NCw0LjkwNjE4MTk0IDI1LjcxNDMxNjQsMTAuNTcxMzY3MyBaIE01NC4wMDAwNjQ0LDI1Ljk5OTk1NzEgQzU0LjAwMDA2NDQsMzQuNTE3ODI0NCA0Ny4wODkzNDE4LDQxLjQyODU0NjkgMzguNTcxNDc0Niw0MS40Mjg1NDY5IEMzMC4wNTM2MDczLDQxLjQyODU0NjkgMjMuMTQyODg0NywzNC41MTc4MjQ0IDIzLjE0Mjg4NDcsMjUuOTk5OTU3MSBDMjMuMTQyODg0NywxNy40ODIwODk4IDMwLjA1MzYwNzMsMTAuNTcxMzY3MyAzOC41NzE0NzQ2LDEwLjU3MTM2NzMgQzQ3LjA4OTM0MTgsMTAuNTcxMzY3MyA1NC4wMDAwNjQ0LDE3LjQ4MjA4OTggNTQuMDAwMDY0NCwyNS45OTk5NTcxIFogTTc3LjE0Mjk0OTEsMzUuMDQwMTQ2NCBDNzcuMTQyOTQ5MSwzOS40OTk5NzMyIDczLjM2NjE1ODksNDEuNDI4NTQ2OSA2OS4zNDgyOTcsNDEuNDI4NTQ2OSBMNjMuOTY0MzYyLDQxLjQyODU0NjkgQzYxLjI3MjM5NDUsMzguMjE0MjU3NCA1Ny40OTU2MDQzLDM2LjQwNjIxOTUgNTMuMzE3MDI3OCwzNi4yODU2ODM2IEM1NS40MDYzMTYsMzMuMjcyMjg3MiA1Ni41NzE0OTYsMjkuNjU2MjExNCA1Ni41NzE0OTYsMjUuOTk5OTU3MSBDNTYuNTcxNDk2LDI1LjExNjAyNzUgNTYuNDkxMTM4OCwyNC4yMzIwOTc4IDU2LjM3MDYwMjksMjMuMzQ4MTY4MiBDNTguMDk4MjgzNSwyMy45NTA4NDc1IDU5Ljg2NjE0MjgsMjQuMjcyMjc2NSA2MS43MTQzNTkzLDI0LjI3MjI3NjUgQzY3LjEzODQ3MjksMjQuMjcyMjc2NSA3MS4zMTcwNDkzLDIwLjg1NzA5MzggNzIuMTYwODAwMywyMC44NTcwOTM4IEM3Ny4yNjM0ODUsMjAuODU3MDkzOCA3Ny4xNDI5NDkxLDMxLjc4NTY3ODMgNzcuMTQyOTQ5MSwzNS4wNDAxNDY0IFogTTcyLjAwMDA4NTgsMTAuNTcxMzY3MyBDNzIuMDAwMDg1OCwxNi4yMzY1NTI2IDY3LjM3OTU0NDYsMjAuODU3MDkzOCA2MS43MTQzNTkzLDIwLjg1NzA5MzggQzU2LjA0OTE3NCwyMC44NTcwOTM4IDUxLjQyODYzMjcsMTYuMjM2NTUyNiA1MS40Mjg2MzI3LDEwLjU3MTM2NzMgQzUxLjQyODYzMjcsNC45MDYxODE5NCA1Ni4wNDkxNzQsMC4yODU2NDA3MTcgNjEuNzE0MzU5MywwLjI4NTY0MDcxNyBDNjcuMzc5NTQ0NiwwLjI4NTY0MDcxNyA3Mi4wMDAwODU4LDQuOTA2MTgxOTQgNzIuMDAwMDg1OCwxMC41NzEzNjczIFoiIGZpbGw9IiNGRkZGRkYiPjwvcGF0aD4gIDwvZz48L3N2Zz4=)",
            }
          }
        />
        <div
          className="label"
        >
          PEOPLE
        </div>
      </a>
    </li>
    <li>
      <a
        className="NavButton"
        href="/projects"
        onClick={[Function]}
        style={undefined}
      >
        <div
          className="icon"
          style={
            Object {
              "backgroundImage": "url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgNjcgNTciIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+ICA8ZyBzdHJva2U9Im5vbmUiIHN0cm9rZS13aWR0aD0iMSIgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj4gICAgPHBhdGggZD0iTTY2Ljg1NzIyMjYsMTkuNzE0MjQxMyBDNjYuODU3MjIyNiwxNC43NzIyNzExIDYyLjc5OTE4MiwxMC43MTQyMzA1IDU3Ljg1NzIxMTgsMTAuNzE0MjMwNSBMMzAuODU3MTc5NiwxMC43MTQyMzA1IEwzMC44NTcxNzk2LDkuNDI4NTE0NzIgQzMwLjg1NzE3OTYsNC40ODY1NDQ1NCAyNi43OTkxMzkxLDAuNDI4NTAzOTkgMjEuODU3MTY4OSwwLjQyODUwMzk5IEw5LjAwMDAxMDczLDAuNDI4NTAzOTkgQzQuMDU4MDQwNTUsMC40Mjg1MDM5OSAwLDQuNDg2NTQ0NTQgMCw5LjQyODUxNDcyIEwwLDQ3Ljk5OTk4OTMgQzAsNTIuOTQxOTU5NCA0LjA1ODA0MDU1LDU3IDkuMDAwMDEwNzMsNTcgTDU3Ljg1NzIxMTgsNTcgQzYyLjc5OTE4Miw1NyA2Ni44NTcyMjI2LDUyLjk0MTk1OTQgNjYuODU3MjIyNiw0Ny45OTk5ODkzIEw2Ni44NTcyMjI2LDE5LjcxNDI0MTMgWiIgZmlsbD0iI0ZGRkZGRiI+PC9wYXRoPiAgPC9nPjwvc3ZnPg==)",
            }
          }
        />
        <div
          className="label"
        >
          PROJECTS
        </div>
      </a>
    </li>
    <li>
      <div
        className="NavButtons"
      >
        <div
          className="userMenu"
        >
          <div
            className="userMenuMenu"
          >
            <button
              className="Button Block"
              disabled={false}
              onClick={[Function]}
              type="button"
            >
              Sign out
            </button>
          </div>
          <div
            className="Avatar avatar"
            onClick={[Function]}
            style={
              Object {
                "backgroundImage": "url(http://www.image.jpg)",
              }
            }
          />
        </div>
      </div>
    </li>
  </ul>
</nav>
`;
